### ISSUE TITLE
zabbixctl -Tp not showing all current problems

### ISSUE THREAD

**Issue by dmd** [lasseoe/zabbixctl#14]
See https://github.com/kovetskiy/zabbixctl/issues/30 which @lasseoe commented on 3 years ago. I'd love for this limitation to be solved in this repo!

**Comment by lasseoe** [comment]
Yeah, it's on my to-do list, let me see if I can fix this sooner rather than later :)
Been busy with other things so I've not had the time to do anything with regards to zabbixctl.   There are a bunch of issues that bubble up once I start making substantial changes though.  I'll comment here within the next week or two when I have a rough idea as to how long it will take.
